demos/multigrid/geometric_multigrid.py.rst

```diff
@@ -167,9 +167,16 @@ remembering to tell PETSc to use pivoting in the factorisation. ::
                                                "ksp_monitor": True,
                                                "pmat_type": "aij"})
 
-Next we'll use a schur complement solver, using geometric multigrid to
+Next we'll use a Schur complement solver, using geometric multigrid to
 invert the velocity block. ::
 
+  class Mass(ExplicitSchurPC):
```

Comment:
Can you describe what it is you're doing here?  This comes out of nowhere for the demo, it seems to me.

Reply:
I've added a brief explanation that the Schur complement is spectrally equivalent to the viscosity-weighted mass matrix, and that we will supply it in this way.